README.md

```diff
@@ -1,3 +1,6 @@
+[!WARNING]<br />
+This package is deprecated as of **2024-02-16**. We haven't had need to create a snapshot for our Elasticsearch cluster for years, and we recommend using the [API](https://www.elastic.co/guide/en/elasticsearch/reference/current/snapshots-take-snapshot.html#manually-create-snapshot) Elasticsearch provides directly in the future.
```

Comment:
**chore:** To match the style of other recent deprecation notices, it requires the quote syntax.

```suggestion
> [!WARNING]<br />
> This package is deprecated as of **2024-02-16**. We haven't had need to create a snapshot for our Elasticsearch cluster for years, and we recommend using the [API](https://www.elastic.co/guide/en/elasticsearch/reference/current/snapshots-take-snapshot.html#manually-create-snapshot) Elasticsearch provides directly in the future.
```